src/open_inwoner/openklant/views/contactform.py

```diff
@@ -60,12 +56,45 @@ def get_context_data(self, **kwargs):
         context["has_configuration"] = config.has_form_configuration()
         return context
 
+    def get_result_message(self, success=False):
+        if success:
+            return messages.add_message(
+                self.request, messages.SUCCESS, _("Vraag verstuurd!")
+            )
+        return messages.add_message(
```

Comment:
Can we put the second message in an else-block? So both parts have same indentation level.